src/helpers/regexHelpers.js

```diff
@@ -1,3 +1,6 @@
 export const imageRegex = /https?:\/\/(?:[-a-zA-Z0-9._]*[-a-zA-Z0-9])(?::\d{2,5})?(?:[/?#](?:[^\s"'<>\][()]*[^\s"'<>\][().,])?(?:(?:\.(?:tiff?|jpe?g|gif|png|svg|ico)|ipfs\/[a-z\d]{40,})))/gi;
 
+export const numberWithCommas = number =>
```

Comment:
Can we replace this with `FormattedNumber` altogether or we require to use this format no matter what locale does user use?

Reply:
@Sekhmet So I initially added this because I thought it would be better than having to parse out the different currencies for each amount, in the different transactions. Since some transactions are in STEEM and some are in SBD. But I just pushed up changes based on your suggestions, lmk if you think this way is better with FormattedNumber